exercises/practice/flower-field/.meta/config.json

```diff
@@ -0,0 +1,34 @@
+{
+  "authors": [
```

Comment:
Minesweeper has `[]` for authors and lists everyone as contributors. Unless it's recommended not to do it this way, I think we should do the same. That, or do some git archeology to try and find out who added the exercise :)

Reply:
Sorry, that was a copy and paste error on my part - had intended to paste the contributors as contributors, but got confused in the process 😅